src/lib/REST/Value/ContentTree/LoadNodeExtendedInfoRequest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\AdminUi\REST\Value\ContentTree;
+
+use Ibexa\Rest\Value as RestValue;
+
+class LoadNodeExtendedInfoRequest extends RestValue
```

Comment:
1. final & internal
2. Naming: in most cases this is a Struct, so `LoadNodeExtendedInfoStruct`. Request suggests to me it's related to HTTP Request. But maybe I'm overthinking this. @ibexa/php-dev any POV? :)